properties.schema

```diff
@@ -30,15 +30,28 @@
       "help": "This is the instruction text",
       "translatable": true
     },
+    "_setCompletionOn": {
+      "type": "string",
+      "required": false,
+      "default": "allItems",
+      "title": "Set completion on:",
+      "enum": ["inview", "allItems"],
+      "inputType": {
+        "type": "Select",
+        "options": ["inview", "allItems"]
+      },
+      "validators": [],
+      "help": "Whether completion is based on the learner having viewed all the tabs - or just having viewed the component"
+    },
     "_tabLayout": {
       "type": "string",
       "required": true,
-      "enum": ["left", "top"],
-      "default": "left",
-      "title": "Tab button position",
-      "inputType": {"type": "Select", "options":["left", "top"]},
+      "enum": ["horizontal", "vertical"],
+      "default": "horizontal",
+      "title": "Tab button layout",
+      "inputType": {"type": "Select", "options":["vertical", "horizontal"]},
       "validators": ["required"],
-      "help": "Select whether the tab buttons should be shown on the left or or the top of the content panel"
+      "help": "Whether the tab buttons should display horizontal across the top of the content panel, or vertically along the side of the content panel. By default, component displays `vertical` layout for medium screen size and below."
```

Comment:
Just to point out that backticks <code>`</code> have no effect in the help text.